rl_algorithms/acer/buffer.py

```diff
@@ -0,0 +1,49 @@
+# TODO : Move to common buffer
+
+from collections import deque
+import random
+
+import torch
+
+from rl_algorithms.common.abstract.buffer import BaseBuffer
+
+
+class ReplayMemory(BaseBuffer):
```

Comment:
Please use existing replay buffer instead making new one.

Reply:
We need to store sequential transition to train with ACER. We need to make memory for ACER.